RPIPs/RPIP-46.md

```diff
@@ -177,7 +177,7 @@ A few details about the reasoning behind the spec:
 - The [revenue share vote](#revenue-share-vote) is intended to allow the main body of the tokenomics to move forward, while allowing more time to get information about our options here before choosing a path
 
 ### RPL issuance rewards and inflation
-As the core value capture is no longer based on a minimum RPL requirement, Saturn 1 removes the minimum to receive RPL issuance rewards. This "cliff" has been an extremely poor piece of UX and anecdotally led to many operators exiting validators. Note that the withdrawal limits are not updated; those had previously aligned with the end of the linear region, and they still do. With Saturn 2, RPL issuance rewards stop entirely. At this point, the bond curve should be creating sufficient attraction that we believe we can get to this desired end state more smoothly.
+As the core value capture is no longer based on a minimum RPL requirement, Saturn 1 removes the minimum to receive RPL issuance rewards. This "cliff" has been an extremely poor piece of UX and anecdotally led to many operators exiting validators. Note that the withdrawal limits are not updated; those had previously aligned with the end of the linear region, and they still do. With Saturn 2, RPL issuance rewards stop entirely. At this point, the bond curve should be creating sufficient attraction that we believe we can get to this desired end state more smoothly. <- [what does the "desired end state" refer to here? I'm not sure I understand what this sentence is trying to highlight"]
```

Comment:
+1, last sentence doesn't quite make sense to me.

Reply:
the desired end state is no issuance. this is tryna say that in Saturn 1 we still need some to keep things attractive for NOs, and that in Saturn 2 that's no longer the case. too late for my brain to think of pretty words for that.

Reply:
Maybe:
> With Saturn 2, RPL issuance rewards stop entirely. The bond curve in Saturn 2 should allow voter share to provide sufficient incentive to stake RPL, but the continuation of RPL issuance rewards under Saturn 1 should help facilitate a smoother transition to this end state

Reply:
Maybe don't call it an end state? Otherwise Samus's wording feels good.

Reply:
Ok, howbout:
> With Saturn 2, RPL issuance rewards stop entirely. Saturn 1 will continue to provide RPL issuance rewards to help facilitate a smoother transition, but with Saturn 2 the bond curve alone should allow voter share to provide sufficient incentive to stake RPL.

Reply:
:rocket: